internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -82,89 +88,212 @@ var Manifest = &appsv1.DaemonSet{
+								// TODO(jaosorior): Use a more restricted selinux type
+								Type: "spc_t",
+							},
 						},
-					},
-					SecurityContext: &v1.SecurityContext{
-						AllowPrivilegeEscalation: &falsely,
-						ReadOnlyRootFilesystem:   &truly,
-						Capabilities: &v1.Capabilities{
-							Drop: []v1.Capability{"ALL"},
-							Add:  []v1.Capability{"CHOWN", "FOWNER", "FSETID", "DAC_OVERRIDE"},
-						},
-						RunAsUser: &userRoot,
-						SELinuxOptions: &v1.SELinuxOptions{
-							// TODO(jaosorior): Use a more restricted selinux type
-							Type: "spc_t",
+						Resources: v1.ResourceRequirements{
+							Requests: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("32Mi"),
+								v1.ResourceCPU:              resource.MustParse("100m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("10Mi"),
+							},
+							Limits: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("64Mi"),
+								v1.ResourceCPU:              resource.MustParse("250m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("50Mi"),
+							},
 						},
 					},
-					Resources: v1.ResourceRequirements{
-						Requests: v1.ResourceList{
-							v1.ResourceMemory:           resource.MustParse("32Mi"),
-							v1.ResourceCPU:              resource.MustParse("100m"),
-							v1.ResourceEphemeralStorage: resource.MustParse("10Mi"),
+					{
+						Name:  "selinux-shared-policies-copier",
+						Image: "quay.io/jaosorior/selinuxd",
+						// Primes the volume mount under /etc/selinux.d with the
+						// shared policies shipped by selinuxd and makes sure the volume mount
+						// is writable by 2000 in order for the controller to be able to
+						// write the policy files. In the future, the policy files should
+						// be shipped by selinuxd directly.
+						//
+						// The directory is writable by 2000 (the operator writes to this dir) and
+						// readable by root (selinuxd reads the policies and runs as root).
+						// Explicitly allowing root makes sure no dac_override audit messages
+						// are logged even in absence of CAP_DAC_OVERRIDE.
+						Command: []string{"bash", "-c"},
+						Args: []string{`
+						set -x
+
+						chown 2000:0 /etc/selinux.d
+						chmod 750 /etc/selinux.d
+						cp /usr/share/udica/templates/* /etc/selinux.d
+					`},
+						VolumeMounts: []v1.VolumeMount{
+							{
+								Name:      "selinux-drop-dir",
+								MountPath: SelinuxDropDirectory,
+							},
 						},
-						Limits: v1.ResourceList{
-							v1.ResourceMemory:           resource.MustParse("64Mi"),
-							v1.ResourceCPU:              resource.MustParse("250m"),
-							v1.ResourceEphemeralStorage: resource.MustParse("50Mi"),
+						SecurityContext: &v1.SecurityContext{
+							AllowPrivilegeEscalation: &falsely,
+							ReadOnlyRootFilesystem:   &truly,
+							Capabilities: &v1.Capabilities{
+								Drop: []v1.Capability{"ALL"},
+								Add:  []v1.Capability{"CHOWN", "FOWNER", "FSETID", "DAC_OVERRIDE"},
```

Comment:
Why was the DAC_OVERRIDE capability needed again?

Reply:
I..might have copied the capabilities of the other init container blindly. I'll test what is really needed. But yeah, DAC_OVERRIDE might not be since we're not accessing anything the usual UNIX permissions wouldn't allow us to access.

Reply:
Dropped.